bin/git-submodule-rewrite

```diff
@@ -50,6 +50,11 @@ EOF
   request_confirmation "Do you want to proceed?"
 }
 
+function git_version_lte() {
+  GIT_VERSION=$(git version)
+  [ "$1" = $(echo -e "${GIT_VERSION#git version}\n$1" | sort -V | head -n1) ]
```

Comment:
I should probably have mentioned this somewhere, but this was only tested on MacOS.  Unfortunately, sort on MacOS doesn't support a '-V' option for semver sorting.  It's cool to know it exists elsewhere, though, so I'll be using that with gnusort.  I'm not sure what options are available otherwise.  There are a few hacks available here:
http://stackoverflow.com/questions/21394536/how-to-simulate-sort-v-on-mac-osx

Thoughts?